menas/src/main/scala/za/co/absa/enceladus/menas/Application.scala

```diff
@@ -28,12 +28,15 @@ import org.springframework.security.core.context.SecurityContextHolder
 @EnableAsync
 @Configuration
 class Application() {
+  private val DefaultCorePoolSize = 12
```

Comment:
I think capital case constants would be more immediately obvious, though this has prob been discussed over and over already.

Reply:
Yeah, we discussed it on CQC, and fro know agreed that both styles are OK.
Funnily enough what's the difference in Scala between `val` especially if of primitve type and a _"constant"_? 😉